apollo-entity/README.md

```diff
@@ -29,7 +29,7 @@ However, this does not work. Theres nothing handling how the `Person` type is be
 from a `ByteString` which is the payload type Apollo works with.
 
 At this point, you'll start writing some custom middlewares using something like Jackson. This
-library contains exactly those middlewares, ready to use directly with your routes.
+library contains exactly those middleware, ready to use directly with your routes.
```

Comment:
this change made the grammar incorrect....

Reply:
I'm happy as long as it is consistent with the rest of the text. Maybe middleware is the plural form..